lib/pbxProject.js

```diff
@@ -1546,7 +1546,7 @@ function pbxShellScriptBuildPhaseObj(obj, options, phaseName) {
     obj.inputPaths = options.inputPaths || [];
     obj.outputPaths = options.outputPaths || [];
     obj.shellPath = options.shellPath;
-    obj.shellScript = '"' + options.shellScript.replace(/"/g, '\\"') + '"';
+    obj.shellScript = '"' + options.shellScript.replace(/\\/g, '\\\\') + '"';
```

Comment:
Looks like it was overwritten incorrectly. Updated the pull request with the replacing of " as well.